### PR TITLE
update vgs docs

### PR DESCRIPTION
**Pull request by kumarkgosa**
# Description
The documentation for the volume-group-snapshot common docs doesn't have the install method in it :
https://dell.github.io/csm-docs/docs/snapshots/volume-group-snapshots/
We have to jump on the github page to know about the install method or upgrade. Hence, It will be better to have it centralized in the doc portal like any other CSM module.
Also updated v1 docs just to be sure.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/snapshots/volume-group-snapshots/_index.md [hunk 1]
```diff
@@ -9,12 +9,55 @@ Description: >
...
+kubectl create -f deploy/kubernetes/snapshot-controller
+```
+
+### 2. Install VGS CRD
+
+```
+IMPORTANT: delete previous v1aplha2 version of CRD and vgs resources created using alpha version.
+	   Snapshots on array will remain if memberReclaimPolicy=retain was used.
+```
+If you want to install the VGS CRD from a pre-generated yaml, you can do so with the following command (run in top-level folder):
```
Comment (rsedlock1958) [inline content/docs/snapshots/volume-group-snapshots/_index.md:35]: Replace "the following" with "this".

File: content/docs/snapshots/volume-group-snapshots/_index.md [hunk 2]
```diff
@@ -9,12 +9,55 @@ Description: >
...
+```
+kubectl apply -f config/crd/vgs-install.yaml
+```
+
+If you want to create your own CRD for installation with Kustomize, then the command `make install` can be used to create and install the Custom Resource Definitions in your Kubernetes cluster.
+
+### 3. Deploy VGS in CSI Driver with Helm Chart Parameters
+The drivers that support Helm chart deployment allow the CSM Volume Group Snapshotter to be _optionally_ deployed 
+by variables in the chart. There is a _vgsnapshotter_ block specified in the _values.yaml_ file of the chart that will look similar the text below by default:
```
Comment (rsedlock1958) [inline content/docs/snapshots/volume-group-snapshots/_index.md:44]: Replace "the text below by default" to "this default text"

File: content/docs/snapshots/volume-group-snapshots/_index.md [hunk 3]
```diff
@@ -9,12 +9,55 @@ Description: >
...
+
+```
+# volume group snapshotter(vgsnapshotter) details
+# These options control the running of the vgsnapshotter container
+vgsnapshotter:
+  enabled: false
+  image: 
+ 
+```
+To deploy CSM Volume Group Snapshotter with the driver, the following changes are required:
```
Comment (rsedlock1958) [inline content/docs/snapshots/volume-group-snapshots/_index.md:54]: Replace "the following" with "these"

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. There are several minor edits required.

File: content/docs/snapshots/volume-group-snapshots/_index.md (hunk 1 above)
Comment (kumarkgosa) [reply content/docs/snapshots/volume-group-snapshots/_index.md]: Done

File: content/docs/snapshots/volume-group-snapshots/_index.md (hunk 2 above)
Comment (kumarkgosa) [reply content/docs/snapshots/volume-group-snapshots/_index.md]: Done

File: content/docs/snapshots/volume-group-snapshots/_index.md (hunk 3 above)
Comment (kumarkgosa) [reply content/docs/snapshots/volume-group-snapshots/_index.md]: Done

Comment (kumarkgosa) [reply content/docs/snapshots/volume-group-snapshots/_index.md]: Done

**Review by JacobGros — APPROVED** [review]

**Review by jooseppi-luna — APPROVED** [review]

**Review by rsedlock1958 — APPROVED** [review]
I have reviewed this. No edits are required.

**Review by mbasha-dell — APPROVED** [review]

**Pull request merged**